tls/s2n_psk.c

```diff
@@ -246,6 +246,17 @@ static S2N_RESULT s2n_psk_write_binder_list(struct s2n_connection *conn, const s
         GUARD_RESULT(s2n_array_get(psk_list, i, (void**) &psk));
         ENSURE_REF(psk);
 
+        /* From https://tools.ietf.org/html/rfc8446#section-4.1.4:
```

Comment:
Nit: Indenting is a bit off here.

Reply:
If you end up fixing this you should adopt the formatting from #2511 🙂